gpu/gpu_info_cuda.c

```diff
@@ -27,6 +28,9 @@ void cuda_init(char *cuda_lib_path, cuda_init_resp_t *resp) {
       {"nvmlDeviceGetVbiosVersion", (void *)&resp->ch.nvmlDeviceGetVbiosVersion},
       {"nvmlDeviceGetBoardPartNumber", (void *)&resp->ch.nvmlDeviceGetBoardPartNumber},
       {"nvmlDeviceGetBrand", (void *)&resp->ch.nvmlDeviceGetBrand},
+      {"nvmlDeviceGetMigMode", (void *)&resp->ch.nvmlDeviceGetMigMode},
```

Comment:
What does the compatibility matrix look like for these MIG capabilities?  From my quick read of the API docs, it seems like it is Linux only, so how does this behave on windows?  The APIs docs aren't clear, but I'm hoping it gives a good not-supported response and this code will behave well on windows in the non-MIG flow.

Reply:
Hmm, yes I'm also hoping that the call returns, and/or the symbol is resolved.  I don't have a means of testing on windows.  This is checked here

if (ret != NVML_SUCCESS && ret != NVML_ERROR_NOT_SUPPORTED) 

and later only attempts MIG stuff if mode == 1

if (currentMode == 0x1)

otherwise it defaults to the normal behavior.